00-Home.md

```diff
@@ -58,6 +58,19 @@ python main.py
 
 The demo should be up and running on `http://localhost:8080/serverapi/`
 
+The default credentials for API endpoints that require authentication are
+```bash
+username : 1
+password : test
+```
+
+
+To add authorised users to your API:
```

Comment:
You should also specify where new users can be added like with CLI arguments and main.py